ludwig/api.py

```diff
@@ -30,6 +30,7 @@
 from pprint import pformat
 from typing import Dict, List, Optional, Tuple, Union
 
+import dask
```

Comment:
Dask is not a required import. We shouldn't be pulling it into api.

Reply:
Fixed.